docs/INSTALL.md

```diff
@@ -1,48 +0,0 @@
-# Installation Instructions
-
-This document covers how to install **pycls** and its dependencies.
-
-- For general information about **pycls**, please see [`README.md`](../README.md)
-
-**Requirements:**
-
-- NVIDIA GPU, Linux, Python3
-- PyTorch, various Python packages; Instructions for installing these dependencies are found below
-
-**Notes:**
-
-- **pycls** does not currently support running on CPU; a GPU system is required
-- **pycls** has been tested with CUDA 9.2 and cuDNN 7.1
-
-## PyTorch
```

Comment:
Can we mention the PyTorch related requirements in Getting Started?

Reply:
Sure, will update the PR